docs/modules/ROOT/pages/implementation-notes.adoc

```diff
@@ -12,5 +12,15 @@ We ensure local access via an https://kubernetes.io/docs/concepts/services-netwo
 
 == OPA Bundle Builder
 
-Each OPA pod runs a container that watches for `ConfigMap` objects labeled `opa.stackable.tech/bundle`.
-All `ConfigMap`s are bundled together in a single `bundle.tar.gz` file that is served to the colocated OPA container.
\ No newline at end of file
+Users can manage the Rego-rules evaluated by an OPA cluster by creating, updating and deleting `ConfigMap` resources.
+
+The role of the bundle builder is to convert these ressources to bundles (`tar.gz` files) and make it available as a HTTP endpoint.
```

Comment:
```suggestion
The role of the bundle builder is to convert these resources to bundles (`tar.gz` files) and make them available via an HTTP endpoint.
```

Reply:
The "ressources" is not fixed yet?

Reply:
I made the same changes locally that's why I couldn't apply your suggestion. It's fixed now.